src/powershell/README.md

```diff
@@ -16,6 +16,8 @@ Installs PowerShell along with needed dependencies. Useful for base Dockerfiles
 | Options Id | Description | Type | Default Value |
 |-----|-----|-----|-----|
 | version | Select or enter a version of PowerShell. | string | latest |
+| modules | Optional comma separated list of PowerShell modules to install. | string | - |
```

Comment:
Thanks for being considerate. Once, this PR is merged, an action would autogenerate this doc with similar contents.

Can we revert this to avoid merge conflicts?

Reply:
Sure - all done!